src/assets/i18n/de.json

```diff
@@ -0,0 +1,51 @@
+{
```

Comment:
Do we have internationalization on warranty page? If not let's rip out all but english.